test/e2e/plain_provisioner_test.go

```diff
@@ -594,6 +594,85 @@ var _ = Describe("plain provisioner bundle", func() {
 				}).Should(BeNil())
 			})
 		})
+
+		When("the bundle is backed by a private repository", func() {
+			var (
+				bundle      *rukpakv1alpha1.Bundle
+				secret      *corev1.Secret
+				privaterepo string
```

Comment:
```suggestion
				privateRepo string
```
nit: per Go's camelCase conventions